docs/access.rst

```diff
@@ -118,6 +118,12 @@ For example, if Kolibri is installed and started on a computer with the address
   * You can also use the ``ipconfig`` command on Windows or ``ifconfig`` command on Linux/OSX to find the externally visible IP address of the device running the Kolibri.
 
 
+.. warning::
+  * When limited to using older browsers it may be necessary to **explicitly include** the ``http://`` in front of the IP in the address bar, for example ``http://192.168.8.134:8080``. Many modern browsers may also take the IP address or the local network hostname as a search term and redirect you to Google. In other cases, browsers may try to use the secure **HTTPS** protocol by default and add an **S**  automatically (``httpS://``), which may also cause issues while running Kolibri.
```

Comment:
```suggestion
  * When typing in your server's address in the browser address bar, it may be required to add `http://` in the first part of the address, for instance `http://1.2.3.4:8080` or `http://1.2.3.4:80`. Many modern browsers may understand an IP address or local network hostname as a search term and redirect a user to Google. In other cases, browsers may add httpS:// automatically, which will also cause issues.
```

@radinamatic notice that the original issue pertained some kind of "older browsers" and @jredrejo is referring to current and upcoming browsers. I think it's best to just describe the behavior and add a simple-to-read caution.

If people are setting up Kolibri on Port 80 and configuring their own host names, i.e. `http://kolibri`, they will probably understand the caution, given that they have a bit of technical background required to make a choice of port 80 :)